TOAST/application/controllers/test/User_test.php

```diff
@@ -12,15 +12,15 @@ function __construct()
 		parent::__construct(__FILE__);
 		$this->UserController = new User(true);	//Instantiate class
 	}
-
+/*
```

Comment:
Ah, sorry, my bad. Can you please comment the code inside the function itself though, instead of the entire function? That's a nit, but I think it'd look better.